recipes/sdl_ttf/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
+            ttf_path = os.path.join(self.source_folder, "OpenSans-Bold.ttf")
```

Comment:
Perhaps, we should add the LICENSE.txt file from the font archive?
See https://www.1001fonts.com/open-sans-font.html#license

Reply:
We have this file in several recipes, maybe this license could be added in a shared folder?